department-of-veterans-affairs/caseflow
# checking if sp is being confirmed and making transactions smoother

Resolves #6793 

### Description
Makes sure the same schedule period or the same dates cannot be submitted twice.

### Acceptance Criteria

### Testing Plan

1st test
---------
1. Upload a RO spreadsheet for specific dates
2. Upload the same spreadsheet again for the same dates
3. Submit the first Schedule period's assignments
4. Submit the second schedule period's assignments while the 1st one is still being submitted.
5. Notice the second submission fails with a message

2nd test
---------
1. Upload a RO spreadsheet for specific dates
2. Submit the schedule period's assignments
3. Hit the back button
4. See the message saying "schedule is still being submitted to VACOLS"
